qa/suites/rgw/verify/tasks/s3tests.yaml

```diff
@@ -1,4 +1,6 @@
 tasks:
 - s3tests:
     client.0:
+      force-branch: fixes_related_to_new_csv_parser
+      git_remote: https://github.com/galsalomon66/
```

Comment:
this is pointing to a branch based on s3tests' `master` branch, which may contain newer tests that don't pass against ceph yet

i rebased your s3tests `fixes_related_to_new_csv_parser` branch on top of `ceph-master` instead and pushed to https://github.com/cbodley/s3-tests/commits/fixes_related_to_new_csv_parser. i also updated this s3tests.yaml to point the `git_remote` at my cbodley repo, and pushed that single commit to https://github.com/cbodley/ceph/commits/wip-qa-rgw-s3select-fixes_related_to_new_csv_parser

i've pushed this PR to ceph-ci for builds at https://shaman.ceph.com/builds/ceph/pr-47848/

once packages are built, i'll schedule the teuthology tests using my updated --suite-branch, ie:
> $ teuthology-suite -s rgw -m smithi -c pr-47848 --suite-repo https://github.com/cbodley/ceph.git --suite-branch wip-qa-rgw-s3select-fixes_related_to_new_csv_parser -p 75

now that this change to s3tests.yaml is in that suite-branch, feel free to remove it from this PR so we don't accidentally merge it

Reply:
ok,
Thanks.